_posts/2025-04-09-#31664.md

```diff
@@ -36,3 +115,14 @@ _Notes and questions to follow soon!_
 
 -->
 {% endirc %}
+
+Previous work and discussions to dig deep:
+
+- [PR #23074 attempts to make the fee estimator package aware](https://github.com/bitcoin/bitcoin/pull/23074)
+- [PR #25380 builds on #23074 to just ignore CPFP'd transactions](https://github.com/bitcoin/bitcoin/pull/25380)
+- [PR #30079 is yet another attempt at #25380](https://github.com/bitcoin/bitcoin/pull/30079)
+- [Delving Bitcoin Post: Package-Aware Fee Estimator](https://delvingbitcoin.org/t/package-aware-fee-estimator-post-cluster-mempool/312)
+- [Issue #27995 by Sipa on improving fee estimation](https://github.com/bitcoin/bitcoin/issues/27995)
+- [Blog Post Discussing challenges of fee estimation](https://hackmd.io/@kEyqkad6QderjWKtcBF5Hg/cChallengies-with-estimating-transaction-fees)
+- [Delving Bitcoin Post: Mempool-Based Fee Estimation](https://delvingbitcoin.org/t/mempool-based-fee-estimation-on-bitcoin-core/703)
+- [Empirical Data Analysis of Fee Rate Forecasters for ASAP (next-block) Fee Estimation](https://delvingbitcoin.org/t/empirical-data-analysis-of-fee-rate-forecasters-for-asap-next-block-fee-estimation/1022)
```

Comment:
This is awesome! Maybe move it up to the end of the notes section?

Reply:
Done